sdk/python/symbolchain/SharedKey.py

```diff
@@ -0,0 +1,16 @@
+import hashlib
+
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.kdf.hkdf import HKDF
+
+from .CryptoTypes import SharedKey256
+from .external import ed25519
+
+
+class SharedKey:
+	@staticmethod
+	def _derive_shared_key(other_public_key_bytes, private_key_bytes, info, hashing_algorithm=hashlib.sha512):
```

Comment:
looking again, is there reason you're passing in bytes vs PublicKey/PrivateKey ?

Reply:
yes, the reason is I need to pass reversed priv key in case of nem 😑